internal/backends/helpers_test.go

```diff
@@ -53,9 +53,10 @@ func testBackends(t *testing.T) map[string]*testBackend {
 		require.NoError(t, err)
 
 		b, err := postgresql.NewBackend(&postgresql.NewBackendParams{
-			URI: testutil.TestPostgreSQLURI(t, context.TODO(), ""),
-			L:   l.Named("postgresql"),
-			P:   sp,
+			URI:        testutil.TestPostgreSQLURI(t, context.TODO(), ""),
+			L:          l.Named("postgresql"),
+			P:          sp,
+			NoContract: true,
```

Comment:
Let's remove that parameter and tests for wraparounds

Reply:
So, we don't need to do `SetRecordID` in the tests anymore? We just test with RecordIDs that the wrapper sets?

Reply:
Yes